RecoEgamma/PhotonIdentification/python/mvaHaloVariable_cfi.py

```diff
@@ -0,0 +1,20 @@
+import FWCore.ParameterSet.Config as cms
+from RecoEgamma.EgammaIsolationAlgos.egammaHBHERecHitThreshold_cff import egammaHBHERecHit
+
+pathToHaloMVATrainingFile = "RecoEgamma/PhotonIdentification/data/beamHaloTaggerID/xgboostToTMVA_BHtagger.root"
```

Comment:
Seems like the `beamHaloTaggerID/xgboostToTMVA_BHtagger.root` file wasn't included, so this PR broke the IB judging from https://github.com/cms-sw/cmssw/pull/37130#issuecomment-1058629919.

Reply:
Thanks! Should be fixed now by  cms-sw/cmsdist#7665